docs/source/getting_started/installation/index.md

```diff
@@ -0,0 +1,45 @@
+## Installing PyMC-Marketing
+
+PyMC-Marketing requires **Python 3.10 or greater**.
+
+Install and activate an environment (e.g. `marketing_env`) with the `pymc-marketing` package from [conda-forge](https://conda-forge.org). It may look something like the following:
+
+```bash
+conda create -c conda-forge -n marketing_env pymc-marketing
+conda activate marketing_env
+```
+
+You can also install the development version of PyMC-Marketing with:
+
+```bash
+pip install git+https://github.com/pymc-labs/pymc-marketing.git
+```
+
+## Notebooks
```

Comment:
The heading here is very mysterious. This is not really about "notebooks" but rather about installing jupyterlab/ipykernel so that the notebooks are runnable in the environment.  It's fine as a section heading, but this heading is then used to populate the index page on the "Getting Started" page and the link just says "Notebooks" and it's not clear what you will get when you click on it.

Reply:
Simplified it in in https://github.com/pymc-labs/pymc-marketing/pull/1023/commits/926f5b545252dc64f0f15a75617390401787d98d